python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -603,16 +603,18 @@ router bgp {{ router_bgp.as }}
 {%         if router_bgp.address_family_evpn.bgp_additional_paths.receive is arista.avd.defined(true) %}
       bgp additional-paths receive
 {%         endif %}
-{%         if router_bgp.address_family_evpn.bgp_additional_paths.send.any is arista.avd.defined(true) %}
-      bgp additional-paths send any
-{%         elif router_bgp.address_family_evpn.bgp_additional_paths.send.backup is arista.avd.defined(true) %}
-      bgp additional-paths send backup
-{%         elif router_bgp.address_family_evpn.bgp_additional_paths.send.ecmp is arista.avd.defined(true) %}
-      bgp additional-paths send ecmp
-{%         elif router_bgp.address_family_evpn.bgp_additional_paths.send.ecmp_limit is arista.avd.defined %}
-      bgp additional-paths send ecmp limit {{ router_bgp.address_family_evpn.bgp_additional_paths.send.ecmp_limit }}
-{%         elif router_bgp.address_family_evpn.bgp_additional_paths.send.limit is arista.avd.defined %}
-      bgp additional-paths send limit {{ router_bgp.address_family_evpn.bgp_additional_paths.send.limit }}
+{%         if router_bgp.address_family_evpn.bgp_additional_paths.send_ecmp_limit is arista.avd.defined and router_bgp.address_family_evpn.bgp_additional_paths.send | arista.avd.default == 'ecmp' %}
+{%             set evpn_add_path = 'bgp additional-paths send ecmp limit ' ~ router_bgp.address_family_evpn.bgp_additional_paths.send_ecmp_limit %}
+{%         elif router_bgp.address_family_evpn.bgp_additional_paths.send_limit is arista.avd.defined and router_bgp.address_family_evpn.bgp_additional_paths.send | arista.avd.default == 'limit' %}
+{%             set evpn_add_path = 'bgp additional-paths send limit ' ~ router_bgp.address_family_evpn.bgp_additional_paths.send_limit %}
+{%         elif router_bgp.address_family_evpn.bgp_additional_paths.send is arista.avd.defined %}
+{%             set evpn_add_path = 'bgp additional-paths send ' ~ router_bgp.address_family_evpn.bgp_additional_paths.send %}
+{%         endif %}
+{%         if router_bgp.address_family_evpn.bgp_additional_paths.prefix_list is arista.avd.defined and evpn_add_path is arista.avd.defined %}
```

Comment:
`prefix-list ` is not applicable here, please check the EOS CLI for each place. I think its applicable only for address-family ipv4.ipv6 only

Reply:
Fixed

Reply:
Removed this from defs and added them in router_bgp at required places only.